lib/pg_saurus/migration/set_role_method.rb

```diff
@@ -13,11 +13,53 @@ class << self
         #
         # @param role [String]
         def set_role(role)
+          if const_defined?("SeedMigrator".to_s) && self.ancestors.include?(SeedMigrator)
```

Comment:
Please remove `.to_s` if it wasn't intentional.

Reply:
Without `.to_s` it raises 'NameError: uninitialized constant'.